Update newsletter and community call sign up to use Odoo survey (#890)
publiccodenet/about: activities/communication/sign-up.md

---
Type: guide
Explains: how to manage sign ups for the newsletter and community calls
---

# Managing newsletter and community call sign up

> We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules that we use are Surveys and Contacts. In Odoo we have one survey called *Newsletter and community calls*, which is the one that configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).

Reviewer: I'd remove "that" after "the modules". I'd replace "one survey

> We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules that we use are Surveys and Contacts. In Odoo we have one survey called *Newsletter and community calls*, which is the one that configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).

Reviewer 2: ```suggestion
We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules we use are Surveys and Contacts. In Odoo we have a survey called *Newsletter and community calls*, which configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).
```

> We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules that we use are Surveys and Contacts. In Odoo we have one survey called *Newsletter and community calls*, which is the one that configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).

Reviewer: ```suggestion
We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules that we use are Surveys and Contacts. In Odoo we have one survey called *Newsletter and community calls*, which is the one that configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).
```

The link for this form can be shared and used wherever we feel that those we are communicating with may be looking for ways to stay in touch with us.

## Update contacts

> When preparing for sending out [invites to community calls](../standard-maintenance/preparing-community-call.md) or the newsletter, first the contacts need to be updated. Check in the notes in the bottom of the survey when it was last updated. In the top right corner of the survey there is a button stating how many answers the survey has. Clicking it reveals all the persons that have signed up and their options. By selecting all the answers that was `Created on` later than in it was last updated and then the download button in the top left corner you get all the answers in one file. With the details from the file, update existing contacts or create new ones. Use the tags, ´Newsletter´, ´Community Call / Foundation', ´Community Call /Standard´ and ´SendValuable´ to store their preferences for being contacted.

Reviewer 2: ```suggestion
When preparing to send out [invitations to community calls](../standard-maintenance/preparing-community-call.md) or the newsletter, first the contacts need to be updated. Check in the notes in the bottom of the survey when it was last updated. In the top right corner of the survey there is a button stating how many answers the survey has. Clicking it reveals all the persons that have signed up and their options. By selecting all the answers that were `Created on` later than when it was last updated and then the download button in the top left corner you get all the answers in one file. With the details from the file, update existing contacts or create new ones. Use the tags ´Newsletter´, ´Community Call / Foundation', ´Community Call /Standard´ and ´SendValuable´ to store their preferences for being contacted.
```
